Formula/pdfpc.rb

```diff
@@ -18,26 +18,21 @@ class Pdfpc < Formula
     sha256 x86_64_linux:   "7f8c4bf4f879d5785c7c0832ca121e93742d82f7c03a67e3f1648028393a7d55"
   end
 
-  head do
-    url "https://github.com/pdfpc/pdfpc.git", branch: "master"
-
-    depends_on "discount"
-    depends_on "json-glib"
-    depends_on "libsoup@2"
-    depends_on "qrencode"
-
-    on_linux do
-      depends_on "webkitgtk"
-    end
-  end
-
   depends_on "cmake" => :build
   depends_on "vala" => :build
+  depends_on "discount"
   depends_on "gst-plugins-good"
   depends_on "gtk+3"
+  depends_on "json-glib"
   depends_on "libgee"
   depends_on "librsvg"
+  depends_on "libsoup@2"
```

Comment:
raise an issue for this in the upstream, https://github.com/pdfpc/pdfpc/issues/664